src/helpers/tracker.js

```diff
@@ -0,0 +1,6 @@
+import MatomoTracker from "@jonkoops/matomo-tracker"
+
+export const tracker = new MatomoTracker({
+  urlBase: "https://www.jankaritech.com/piwik",
```

Comment:
does this page shows the stats?

Reply:
yes, that is where the URL of the tracker